config.json.example

```diff
@@ -94,8 +94,8 @@
     },
     "ITUNES_SHARED_SECRET": "aaaabbbbccccddddeeeeffff00001111",
     "EMAILS" : {
-        "COMMUNITY_MANAGER_EMAIL" : "community@example.com",
-        "TECH_ASSISTANCE_EMAIL" : "tech@example.com",
-        "PRESS_ENQUIRY_EMAIL" : "press@example.com"
+        "COMMUNITY_MANAGER_EMAIL" : "leslie@habitica.com",
+        "TECH_ASSISTANCE_EMAIL" : "admin@habitica.com",
+        "PRESS_ENQUIRY_EMAIL" : "leslie@habitica.com"
```

Comment:
I'm not a fan of this. The example file isn't supposed to hold production configs, but placeholders for local development. And if we're checking to see if strings use appropriate variables, it's better that these be something we'd never have hardcoded in production. With it this way, in a screenshot there's no way to distinguish between a string that was written with "email leslie@habitica.com" vs. one that's written with "email <%= communityManager %>" or what have you -- the configuration would make them look identical.